packages/backend-plugin-api/report.api.md

```diff
@@ -24,6 +25,76 @@ import { Readable } from 'stream';
 import { Request as Request_2 } from 'express';
 import { Response as Response_2 } from 'express';
 
+// @public
+export type AuditorEvent = [
+  message: string,
+  meta: {
+    actor: AuditorEventActorDetails;
+    eventName: string;
+    stage: string;
+    meta?: JsonObject;
+    request?: AuditorEventRequest;
+  } & AuditorEventStatus,
+];
+
+// @public (undocumented)
+export type AuditorEventActorDetails = {
+  actorId?: string;
+  ip?: string;
+  hostname?: string;
+  userAgent?: string;
+};
+
+// @public
+export type AuditorEventArgs<T extends JsonObject> = {
+  message: string;
+  eventName: string;
```

Comment:
Let's get some naming patterns going here, although more importantly we'll want to have a bit of usage in existing plugins to get a feel for how the API is used.